### ISSUE TITLE
Support for Umbraco 7.6.x + Umbraco Deploy?

### ISSUE THREAD

**Issue by hfloyd** [umbraco/Umbraco.Courier.Contrib#35]
I just had my Umbraco Cloud site updated to Umbraco 7.6.3 + Umbraco Deploy, and the Resolvers no longer work. If I include the Nuget Package / .DLL, I get a YSOD error:

> Could not load all types from "Umbraco.Courier.Contrib.Resolvers, Version=1.0.39.25535, Culture=neutral, PublicKeyToken=null" due to LoaderExceptions, skipping:
> . System.IO.FileNotFoundException: Could not load file or assembly 'Umbraco.Courier.DataResolvers, Version=1.2.5210.19577, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.

What is the strategy for supporting Umbraco Deploy?

Thanks!

**Comment by Shazwazza** [comment]
Courier and Deploy should not be used together, that will not be supported. Deploy replaces Courier entirely so they should not be operating side by side.

**Comment by hfloyd** [comment]
Hi @Shazwazza Yes, I figured that a site wouldn't use both Courier and Deploy... So my question was whether the idea of a community resource of DataResolvers for Deploy would be available, or if this repo would have two branches... or whatever. 

After I posted here, I did come across this: https://github.com/umbraco/Umbraco.Deploy.ValueConnectors which seems to indicate that "ValueConnectors" are the Deploy equivalent to Courier's "DataResolvers". The GitHub project, however is not an active repo - with code which can be installed into a site, but rather a collection of examples - copies of the DataConnectors currently compiled into Deploy by default.

I am wondering if HQ can comment on the strategy for managing these custom implementations going forward.
